pkg/resources/data.go

```diff
@@ -640,6 +642,23 @@ func GetCSIMigrationFeatureGates(cluster *kubermaticv1.Cluster) []string {
 				}
 			}
 		}
+	} else if !ccm && gte23.Check(cluster.Status.Versions.ControlPlane.Semver()) {
```

Comment:
Could this explode when no cluster status exists yet (see #9420 , #9427)?

Reply:
I guess it could. Let me quickly push a fix.

Reply:
@xrstf Fixed, PTAL.